start-firestore

```diff
@@ -6,8 +6,14 @@ if [[ -z "${FIRESTORE_PROJECT_ID}" ]]; then
   exit 1
 fi
 
+# Check user environment variable
+if [[ -z "${FIRESTORE_PORT}" ]]; then
+  FIRESTORE_PORT=8088
+  echo "Missing FIRESTORE_PORT environment variable. Set default one to ${FIRESTORE_PORT}" >&2
+fi
+
 # Config gcloud project
 gcloud config set project ${FIRESTORE_PROJECT_ID}
```

Comment:
gcloud config set project ${FIRESTORE_PROJECT_ID}
                          ^---------------------^ SC2086: Double quote to prevent globbing and word splitting.